packages/react-native/ios/BacktraceReactNative.mm

```diff
@@ -15,7 +15,7 @@ @implementation BacktraceReactNative
     }
     instance = [[BacktraceCrashReporter alloc] initWithBacktraceUrl:submissionUrl andDatabasePath: databasePath andAttributes: attributes andOomSupport:TRUE andAttachments:attachmentPaths];
     [instance start];
-    return instance;
+    return nil;
```

Comment:
do we need to return something from the function?

Reply:
nope, the return value is not used at all.